modules/core/src/main/java/org/apache/ignite/internal/processors/rest/handlers/cluster/GridBaselineCommandResponse.java

```diff
@@ -0,0 +1,160 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.processors.rest.handlers.cluster;
+
+import java.io.Externalizable;
+import java.io.IOException;
+import java.io.ObjectInput;
+import java.io.ObjectOutput;
+import java.util.Collection;
+import org.apache.ignite.cluster.BaselineNode;
+import org.apache.ignite.internal.util.typedef.internal.S;
+import org.apache.ignite.internal.util.typedef.internal.U;
+
+import static java.util.stream.Collectors.toList;
+
+/**
+ * Result for baseline command.
+ */
+public class GridBaselineCommandResponse implements Externalizable {
+    /** */
+    private static final long serialVersionUID = 0L;
+
+    /** Cluster state. */
+    private boolean active;
+
+    /** Current topology version. */
+    private long topVer;
+
+    /** Current baseline nodes. */
+    private Collection<String> baseline;
+
+    /** Current server nodes. */
+    private Collection<String> srvs;
+
+    /**
+     * @param nodes Nodes to process.
+     * @return Collection of consistentIds.
+     */
+    private static Collection<String> consistentIds(Collection<? extends BaselineNode> nodes) {
+        return nodes.stream().map(n -> String.valueOf(n.consistentId())).collect(toList());
```

Comment:
How likely "n.consistentId()" will be null? I doubt that it's possible.

Reply:
@ibessonov  It is not to avoid "null", but because consistentId declared as Object, but REST can operate only with Strings.

Reply:
I see that String is required. I was wondering about "String.valueOf" usage instead of simple "*.toString()".

Reply:
I checked and can not find the case when it can be null.